### PR TITLE
Change the default value of CleanPodPolicy to None in comments.

### PR DESCRIPTION
**Pull request by Syulin7**
Set None as default.

Fixes: https://github.com/kubeflow/training-operator/issues/1753

### CODE REVIEW

**Comment by Syulin7** [comment]
build/test was failed:
```
Run golangci-lint run --config=linter_config.yaml ./...
level=warning msg="[runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused."
level=warning msg="[runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused."
Error: pkg/controller.v1/common/job_test.go:194:2: SA4017: AddDate doesn't have side effects and its return value is ignored (staticcheck)
	oneDayAgo.AddDate(0, 0, -1)
	^
Error: Process completed with exit code 1.
```
@johnugeorge @tenzen-y  Do you know the reason why it happened, or can you help make it retry.

**Comment by tenzen-y** [comment]
I guess the behavior of staticcheck changed since we always install the latest golangci-lint like the following:

https://github.com/kubeflow/common/blob/9ec55d141f90faaf52fd6df271e987e5a6781945/.github/workflows/build.yml#L42

So we need to fix the lint error. Can you fix the error?

**Comment by Syulin7** [comment]
OK, I'll give it a try.

**Comment by Syulin7** [comment]
@tenzen-y  I fixed the error, can you help make it retry.

**Comment by tenzen-y** [comment]
> @tenzen-y I fixed the error, can you help make it retry.

Thanks for fixing the error. However, I don't have permission to approve the CI.

@kubeflow/wg-training-leads Can you approve CI?

**Comment by Syulin7** [comment]
@johnugeorge @terrytangyuan PTAL, thanks.

**Comment by johnugeorge** [comment]
Thanks @Syulin7 

/lgtm
/approve

**Comment by Syulin7** [comment]
@terrytangyuan @gaocegege @Jeffwan PTAL, thanks.

File: pkg/apis/common/v1/types.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ const (
 // active.
 type RunPolicy struct {
 	// CleanPodPolicy defines the policy to kill pods after the job completes.
-	// Default to Running.
+	// Default to None.
```
[the remark this answers is not in the document]
Comment (Syulin7) [reply pkg/apis/common/v1/types.go:182]: the code changes in training operator, PR: https://github.com/kubeflow/training-operator/pull/1754. and there are only comments in the common library. The generation of the CRD in the training operator will use it.

**Comment by johnugeorge** [comment]
@terrytangyuan Can you merge it as well?

**Review by terrytangyuan — APPROVED** [review]
/lgtm
/approve

**Pull request merged**